### ISSUE TITLE
Notes within Notes

### ISSUE THREAD

**Issue by Bizzet** [streetwriters/notesnook#2031]
### What problem are you facing?

I get frustrated that I cannot easily print multiple notes at once.

### What's the solution you'd like to see in Notesnook?

I would like to be able to add the content from other notes inside of a note.

I would imagine this would look similar to the Code feature but instead of code syntax, it would instead have the title and text of another note.

### Example
**`——— Inline Note Title —————————————-———————————— `**
`The text of another note would be underneath the note title. This is the best example I can make using Markdown but you should be able to get the idea`

Basically it would work like this:

- User is writing a note 
- User selects the + content menu within the editor
- User selects **Inline Note** content type
- A pop-up to select a note appears similar to the pop-up that appears when adding a note to a notebook
- The selected note content will now be embedded into the note that was being edited

### What alternatives have you considered?

Copy and paste every note into one note
OR
Export all notes and use Microsoft Word to concatenate the notes then print using Microsoft Word
